tensorflow/core/kernels/crop_and_resize_op.cc

```diff
@@ -228,61 +230,58 @@ struct CropAndResize<CPUDevice, T> {
           continue;
         }
 
-        const float height_scale =
-            (crop_height > 1)
-                ? (y2 - y1) * (image_height - 1) / (crop_height - 1)
-                : 0;
-        const float width_scale =
-            (crop_width > 1) ? (x2 - x1) * (image_width - 1) / (crop_width - 1)
-                             : 0;
-
-        for (int y = 0; y < crop_height; ++y) {
-          const float in_y = (crop_height > 1)
-                                 ? y1 * (image_height - 1) + y * height_scale
-                                 : 0.5 * (y1 + y2) * (image_height - 1);
-          if (in_y < 0 || in_y > image_height - 1) {
-            for (int x = 0; x < crop_width; ++x) {
-              for (int d = 0; d < depth; ++d) {
-                crops(b, y, x, d) = extrapolation_value;
-              }
-            }
-            continue;
+        if (method_name == "bilinear") {
+          std::vector<CachedInterpolation> xs;
+          std::vector<CachedInterpolation> ys;
+          int min_ix, max_ix, min_iy, max_iy;
+          // TODO: Should the half_pixel_centers be added to this function?
```

Comment:
Just wanted to highlight whether half-pixel center logic would be useful here

Reply:
Whatever we do here, it has to be consistent with the behavior in resize_v2.

@johnpjf FYI

Reply:
This one (crop and resize) wasn't actually updated yet. My plan was to make bilinear_resize accept a translate arg, then implement crop and resize with that.